pkg/oc/bootstrap/docker/run_self_hosted.go

```diff
@@ -115,6 +115,10 @@ func (c *ClusterUpConfig) StartSelfHosted(out io.Writer) error {
 		"OPENSHIFT_CONTROLLER_MANAGER_CONFIG_HOST_PATH": configDirs.openshiftControllerConfigDir,
 		"NODE_CONFIG_HOST_PATH":                         configDirs.nodeConfigDir,
 		"KUBEDNS_CONFIG_HOST_PATH":                      configDirs.kubeDNSConfigDir,
+		"OPENSHIFT_PULL_POLICY":                         c.defaultPullPolicy,
+		"OPENSHIFT_IMAGE":                               c.openshiftImage(),
+		"OPENSHIFT_TSB_IMAGE":                           c.ImageTemplate.ExpandOrDie("origin-template-service-broker"),
+		"OPENSHIFT_WEBCONSOLE_IMAGE":                    c.ImageTemplate.ExpandOrDie("web-console"),
```

Comment:
This looks off.  It should be in `oc cluster add`, right?

Reply:
err, right...